up42/host.py

```diff
@@ -4,3 +4,7 @@
 def endpoint(path: str):
     """Gets endpoint url based on its path."""
     return f"https://api.up42.{DOMAIN}{path}"
+
+
+def token_endpoint():
+    return f"https://auth.up42.{DOMAIN}/realms/public/protocol/openid-connect/token"
```

Comment:
```suggestion
    return f"https://auth.up42.com/realms/public/protocol/openid-connect/token"
```
NIT Why do we need `DOMAIN` at all? I see it always being `com`. If we want to keep a constant we could also make it `https://auth.up42.com`.